contents/blog/what-you-need-to-know-about-open-source-analytics.md

```diff
@@ -0,0 +1,53 @@
+
+Product analytics is useful for many different teams within an organization, from marketing to finance, and can be used in any type of product, from [games](/blog/building-the-future-of-game-analytics-pureskill) to [productivity software](/customers/saga). Teams can use product analytics to answer questions such as: 
+
+- What content should I create? 
+- What should I change to make my site more accessible and navigable? 
+- How can I improve my conversion rate or increase my earnings? 
+- How do users discover my product?
+- How do users react to our pricing?
+
+## What is open source software?
+Open source is a licensing model which enables anyone to see and modify the source code within the software. This isn’t the case for most software, which is closed source. There are a variety of open-source licenses, but most open source projects are also distributed for free - this is called Free and Open Source Software (FOSS). 
+
+PostHog is an example of an open-source project, as anyone can access our source code via our [GitHub repo](https://github.com/PostHog/posthog) and we offer a [FOSS version](https://github.com/PostHog/posthog-foss) of PostHog under a permissive MIT license. This means that the only restriction is that a license and copyright notice must be included with all versions of the software.
+
+## Why is being open-source important in product analytics?
+PostHog’s mission is to help more companies build successful products and the decision to be open source is part of that. We believe we need to change how product analytics works and that the open-source model will enable faster, better, community-driven progress. 
+
+For example, teams using closed-source analytics tools are often limited on which software they can use or integrate with. An open-source approach removes this issue because teams can add to our source code directly and create new integrations between platforms by themselves. Many popular PostHog plugins were developed by the community, for example. 
+
+An open-source product analytics platform also offers other advantages, such as the ability to influence the product roadmap or enabling users to fix bugs themselves instead of waiting for customer support. If PostHog were to make decisions which the community disagreed with then the users can express that more directly. 
+
+Being open-source doesn't just offer additional protection to the community - it also better protects the community's users too. This is because open-source platforms can be self-hosted on a team's existing infrastructure, unlike closed-source solutions. Teams which use these closed-source platforms have to share any data they want to analyze with third-parties, such as MixPanel and Amplitude. This data therefore becomes more vulnerable to security breaches, but with an open-source solution such as PostHog the data never has to be shared with anyone. 
```

Comment:
Minor comment: I would consider taking this from an angle of "companies now have full control of their data and only they have access", security breaches can seem a bit more aggressive and in the end PostHog Cloud would be susceptible to that too.